go.mod

```diff
@@ -18,13 +18,13 @@ require (
 	github.com/spiffe/spire-api-sdk v1.11.1
 	github.com/stretchr/testify v1.10.0
 	google.golang.org/grpc v1.69.0
-	google.golang.org/protobuf v1.35.2
+	google.golang.org/protobuf v1.36.0
```

Comment:
I think this is the bump that we had to revert in the SDK

Reply:
Looks like [this change](https://github.com/bufbuild/protoyaml-go/pull/51) fixes the panic we saw.